### DISCUSSION TITLE
Is there any way to add mouse over event to display image on hover over highlighted text?

### DISCUSSION THREAD

**Discussion by deven740** [pymupdf/PyMuPDF#2147] [Looking for help]
Hi everyone!

I have a PDF document. I have need to show image on mouse hover over some text I have highlighted and hide image when no mouse is hovering on the text.

Is there any way to achieve this? Thanks in advance.

**Comment by JorjMcKie** [comment]
This would be a feature of your PDF viewer like Acrobat, over which there exists not control. This a thing outside PyMuPDF's scope.

One **_could_** additionally add a link (pointing to nowhere) at places where there exists a highlight annot, that would at least change the cursor to the **hand** symbol.
I know of no way to influence the cursor appearance dynamically given a specific viewer.
